_posts/en/newsletters/2021-01-20-newsletter.md

```diff
@@ -0,0 +1,125 @@
+---
+title: 'Bitcoin Optech Newsletter #132'
+permalink: /en/newsletters/2021/01/20/
+name: 2021-01-20-newsletter
+slug: 2021-01-20-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes posts to the Bitcoin-Dev mailing list
+about payjoin adoption and making hardware wallets compatible with more
+advanced Bitcoin features.  Also included are our regular sections with
+overviews of changes to services and client software, new releases and
+release candidates, and changes to popular Bitcoin infrastructure
+software.
+
+## News
+
+- **Payjoin adoption:** Chris Belcher [posted][belcher payjoin] to the
+  Bitcoin-Dev mailing list a request for people to look for ways to
+  increase [payjoin][topic payjoin] adoption along with a [wiki
+  page][payjoin wiki] tracking the projects that provide either sending
+  or receiving support for payjoin.  One [suggestion][raw payjoin] by
+  Craig Raw was to extend the protocol to allow it to work even when the
+  receiver doesn't operate a server.
+
+- **Making hardware wallets compatible with more advanced Bitcoin features:**
+  Kevin Loaec [started a discussion][loaec hww] on the Bitcoin-Dev
+  mailing list about how hardware wallets could be changed to allow them
+  to handle scripts more complicated than single-sig or multisig.  For
+  example, allowing a hardware wallet to handle in-channel LN payments
+  or payments made from a [vault][topic vaults].  His post does a good
+  job of describing various problems that current hardware wallets can't
+  handle, but he notes that necessary "changes may be very difficult".
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+FIXME:bitschidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding to update Tuesday
+
+- [Bitcoin Core 0.21.0][Bitcoin Core 0.21.0] is a the next major version
+  of this full node implementation and its associated wallet and other
+  software.  Major new features include support for Tor version 3 hidden
+  services using [version 2 `addr` messages][topic addr v2], the optional
```

Comment:
"using version 2 addr messages" seems odd, both on its own and also following "Tor version 3"

maybe write something like "Tor v3 next-generation onion services" or "Tor's Next Generation Onion Services" (as [Tor calls them](https://blog.torproject.org/tors-fall-harvest-next-generation-onion-services)) and/or "using BIP155 `addrv2`" messages and link to BIP155